dataline-integrations/singer/bigquery/destination/build.gradle

```diff
@@ -1,34 +1,51 @@
+import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
+import groovy.json.JsonSlurper
 plugins {
     id 'java'
     id 'com.bmuschko.docker-remote-api' version '6.6.1'
 }
 
 def image = 'dataline/integration-singer-bigquery-destination:dev'
 
+sourceSets {
+    integrationTest {
+        java {
+            srcDir 'src/test-integration/java'
+        }
+        resources {
+            srcDir 'src/test-integration/resources'
+        }
+    }
+}
+test.dependsOn('compileIntegrationTestJava')
```

Comment:
That is the fix to ensure we build the integration tests. @jrhizor if you think there is a better way, let me know.

Reply:
Looks good